metadata-db-app/src/migrations/050_setup_variable_associations_table.clj

```diff
@@ -0,0 +1,60 @@
+(ns migrations.050-setup-variable-associations-table
+  (:require
+   [config.mdb-migrate-helper :as h]))
+
+(def ^:private variable-assocs-column-sql
+  "id NUMBER,
+  concept_id VARCHAR(255) NOT NULL,
+  native_id VARCHAR(1500) NOT NULL,
+  metadata BLOB NOT NULL,
```

Comment:
This is a good observation. Our metadata field value is the gziped bytes of the original metadata and it is not NULL for empty string; in other words, we will never have NULL or empty string as value for our metadata field. This is true for all concept types in metadata-db, not just variable associations. 

I am guessing @jasongilman did it for a reason. If you want, we can file a separate issue to look into adding empty string handling in our utility lib.